src/docs/platforms/python/chalice.mdx

```diff
@@ -0,0 +1,53 @@
+---
+draft: false
+categories: []
+toc: true
+title: Chalice
+sidebar_order: 2
+tags: []
+---
+The Chalice integration adds support for the [Chalice aws](https://aws.github.io/chalice/).
```

Comment:
Please use the same formatting as in https://github.com/getsentry/sentry-docs/blob/master/src/docs/platforms/python/pyramid.mdx

1. "new in version" phrasing and italics (and in parenthesis)